Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -127,6 +127,25 @@ div.bocList table.bocListTable td.bocListTitleCell > span
   padding: calc(var(--remotion-themed-spacing) / 2) var(--remotion-themed-spacing);
 }
 
+div.bocList table.bocListTable th.bocListTitleCell > a > img,
+div.bocList table.bocListTable th.bocListTitleCell > span > img
+{
+  width: 16px;
+  height: 16px;
+  margin-right: var(--remotion-themed-spacing); /* Provides a spacing between icon and text, which is reset below if only an icon is displayed */
+}
+
+div.bocList table.bocListTable th.bocListTitleCell > a > img:has(+ span.screenReaderText),
+div.bocList table.bocListTable th.bocListTitleCell > span > img:has(+ span.screenReaderText)
+{
+  margin-right: 0;
+}
```

Comment:
Simple CSS:
if we do the following, we should be able to get the same effect with more straight-forward markup:
```
div.bocList table.bocListTable th.bocListTitleCell > a > img+span:not(.screenReaderText),
div.bocList table.bocListTable th.bocListTitleCell > span > img+span:not(.screenReaderText)
{
  padding-left: var(--remotion-themed-spacing-small); 
}
```

Actually, we don't even need to test for `:not(.screenReaderText)` on the span since the padding would be invisible anyway.

I used padding instead of margin. It should work. If not, we can stick with margin. (I use padding everywhere else in these cases)

Reply:
I have now solved this problem using a `gap` in the flex box definition above :)